src/nbl/video/IPhysicalDevice.cpp

```diff
@@ -6,6 +6,13 @@ namespace nbl::video
 IPhysicalDevice::IPhysicalDevice(core::smart_refctd_ptr<system::ISystem>&& s, core::smart_refctd_ptr<asset::IGLSLCompiler>&& glslc) :
     m_system(std::move(s)), m_GLSLCompiler(std::move(glslc))
 {
+    // Make sure everything is set to 0 or false in props/features
+    memset(&m_properties, 0, sizeof(SProperties));
+    memset(&m_features, 0, sizeof(SFeatures));
+    memset(&m_memoryProperties, 0, sizeof(SMemoryProperties));
+    memset(&m_linearTilingUsages, 0, sizeof(SFormatImageUsage));
+    memset(&m_optimalTilingUsages, 0, sizeof(SFormatImageUsage));
+    memset(&m_bufferUsages, 0, sizeof(SFormatBufferUsage));
```

Comment:
howabout nice default initialized values for the structs instead?

Reply:
Ok